2024-02-15 18:47:19,263 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 18:47:19,286 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 18:47:19,286 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 18:47:20,164 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 18:47:20,183 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 18:47:20,183 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 18:50:07,195 DEBUG: verify_token: 4878f4ad0c780de89867e3c11ab010e4 [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:24]
2024-02-15 18:50:07,195 DEBUG: verify_token: 4878f4ad0c780de89867e3c11ab010e4 [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:24]
2024-02-15 18:50:55,659 DEBUG: credentials: Pep1 [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:13]
2024-02-15 18:50:55,659 DEBUG: credentials: Pep1 [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:13]
2024-02-15 18:50:55,660 DEBUG: auth user: OrderedDict({'id': 59, 'name': 'Pep1', 'email': 'oscasdar@oscar.es', 'role': 'wanner', 'created': datetime.datetime(2024, 1, 19, 15, 28, 18, 633692), 'updated': datetime.datetime(2024, 2, 15, 16, 39, 48), 'email_token': '0mMyIq-VZlakd7Xto9R7X2BYDvE', 'verified': 'false', 'token': '4878f4ad0c780de89867e3c11ab010e4', 'token_expiration': datetime.datetime(2024, 2, 15, 17, 39, 48, 84770)}) [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:14]
2024-02-15 18:50:55,660 DEBUG: auth user: OrderedDict({'id': 59, 'name': 'Pep1', 'email': 'oscasdar@oscar.es', 'role': 'wanner', 'created': datetime.datetime(2024, 1, 19, 15, 28, 18, 633692), 'updated': datetime.datetime(2024, 2, 15, 16, 39, 48), 'email_token': '0mMyIq-VZlakd7Xto9R7X2BYDvE', 'verified': 'false', 'token': '4878f4ad0c780de89867e3c11ab010e4', 'token_expiration': datetime.datetime(2024, 2, 15, 17, 39, 48, 84770)}) [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:14]
2024-02-15 18:50:56,701 DEBUG: Token:dc9ed70b83ddb5ecdb684e3892b4d76c [in E:\2DAW\M12\M12-Projecte1\app\api\tokens.py:10]
2024-02-15 18:50:56,701 DEBUG: Token:dc9ed70b83ddb5ecdb684e3892b4d76c [in E:\2DAW\M12\M12-Projecte1\app\api\tokens.py:10]
2024-02-15 18:51:00,620 DEBUG: verify_token: dc9ed70b83ddb5ecdb684e3892b4d76c [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:24]
2024-02-15 18:51:00,620 DEBUG: verify_token: dc9ed70b83ddb5ecdb684e3892b4d76c [in E:\2DAW\M12\M12-Projecte1\app\api\helper_auth.py:24]
2024-02-15 19:34:15,139 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:34:15,158 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:34:15,158 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:02,924 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:02,943 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:02,943 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:15,391 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:15,410 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:15,410 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:16,227 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:16,247 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:36:16,247 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:46:41,609 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:46:41,609 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:46:47,094 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:46:47,094 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:46:56,613 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:46:56,635 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:46:56,635 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:46:57,422 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:46:57,439 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:46:57,439 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:47:01,368 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:47:01,368 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:48:18,033 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:48:18,033 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:48:27,484 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:48:27,484 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:48:29,311 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:48:29,311 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:49:35,584 DEBUG: Usuari User1e intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:49:35,584 DEBUG: Usuari User1e intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:49:49,819 DEBUG: Usuari Pep intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:49:49,819 DEBUG: Usuari Pep intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:50:07,696 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:50:07,696 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:52:13,418 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:52:13,437 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:52:13,437 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-15 19:55:22,337 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:55:22,337 DEBUG: Usuari User1 intenta autenticar-se [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:61]
2024-02-15 19:55:23,655 INFO: Usuari User1 s'ha autenticat correctament [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:70]
2024-02-15 19:55:23,655 INFO: Usuari User1 s'ha autenticat correctament [in E:\2DAW\M12\M12-Projecte1\app\routes_auth.py:70]
2024-02-16 15:33:18,424 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-16 15:33:18,461 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-16 15:33:18,461 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-16 15:33:19,496 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-16 15:33:19,514 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
2024-02-16 15:33:19,514 INFO: Aplicación iniciada [in E:\2DAW\M12\M12-Projecte1\app\__init__.py:65]
